### ISSUE TITLE
High Resolution Time is erroneously marked as retired

### ISSUE THREAD

**Issue by xfq** [w3c/web-roadmaps#434]
In [Performance and Tuning](https://w3c.github.io/web-roadmaps/mobile/performance.html), the maturity of High Resolution Time is retired, but the [spec](https://www.w3.org/TR/2019/REC-hr-time-2-20191121/) is actually a REC.

**Comment by tidoust** [comment]
Interesting. Level 2 is a Recommendation but level 1 was superseded at the same time. Both levels share the same short name and the W3C API returns the info about the superseded version: `https://api.w3.org/specifications/hr-time/versions/20191121?apikey=[...]`

@deniak Wouldn't it be better to return the info of the active level?

@xfq In the meantime, fix is to use the level 2 shortname `hr-time-2`. I'll create a PR.

**Comment by deniak** [comment]
That's a known issue because we reuse shortname for different specs and at the same time, the shortname is an identifier in the API. `hr-time-1` was initially published under the shortname `hr-time` so the system treats that use case as a shortname change and associate `hr-time` to the level 1.

What I can do is to explicitly set `hr-time-2` superseding `hr-time` so the information will appear on https://api.w3.org/specifications/hr-time. Do you think that will help?

**Comment by tidoust** [comment]
> What I can do is to explicitly set hr-time-2 superseding hr-time so the information will appear on https://api.w3.org/specifications/hr-time. Do you think that will help?

It won't help us in the sense that it's easier for us to simply use the `hr-time-2` URL than to code the logic that follows the superseded link. But that sounds conceptually better to at least detect that the info is not the info we're looking for.

**Comment by deniak** [comment]
The current API routes were designed before we start handling the version-less shortnames. I'm thinking the API should now take these into account either by adding new routes or updating the data the API displays. I'll give it more thoughts and start discussing this with @plehegar, @vivienlacourba and @jean-gui.

In the meantime, I've marked `hr-time` superseded by `hr-time-2`.
